cmd/vcert/utils.go

```diff
@@ -165,6 +166,36 @@ func fillCertificateRequest(req *certificate.Request, cf *commandFlags) *certifi
 		}
 		req.CsrOrigin = certificate.LocalGeneratedCSR
 	}
+
+	if cf.validDays != "" {
+		validDays := cf.validDays
+
+		data := strings.Split(validDays, "#")
+		days, _ := strconv.ParseInt(data[0], 10, 64)
+		hours := days * 24
+
+		req.ValidityHours = int(hours)
+
+		issuerHint := "Specific End Date"
```

Comment:
Empty string would be less confusing here since this value isn't actually used for anything.

Reply:
fixed on most recent commit